docs/setup/configuring-logging.asciidoc

```diff
@@ -0,0 +1,696 @@
+[[logging-configuration]]
+== Configure logging
+
+The {kib} logging system has three main components: _loggers_, _appenders_ and _layouts_. These components allow us to log messages according to message type and level, to control how these messages are formatted and where the final logs will be displayed or stored.
+
+- <<loggers-appenders-layout>>
+- <<log-level>>
+- <<logging-layouts>>
+- <<logger-hierarchy>>
+
+[float]
+[[loggers-appenders-layout]]
+=== Loggers, Appenders and Layouts
+__Loggers__ define what logging settings should be applied to a particular logger.
+
+__<<logging-appenders,Appenders>>__ define where log messages are displayed (eg. stdout or console) and stored (eg. file on the disk).
+
+__<<logging-layouts,Layouts>>__ define how log messages are formatted and what type of information they include.
+
+[float]
+[[log-level]]
+=== Log level
+
+Currently we support the following log levels: _all_, _fatal_, _error_, _warn_, _info_, _debug_, _trace_, _off_.
+
+Levels are ordered, so _all_ > _fatal_ > _error_ > _warn_ > _info_ > _debug_ > _trace_ > _off_.
+
+A log record is being logged by the logger if its level is higher than or equal to the level of its logger. Otherwise, the log record is ignored.
```

Comment:
```suggestion
A log record will be logged by the logger if its level is higher than or equal to the level of its logger. Otherwise, the log record is ignored.
```